### DISCUSSION TITLE
Contributing documentation

### DISCUSSION THREAD

**Discussion by tjcafferkey** [rewindui/rewindui#62] [General]
At the moment there's no clear documentation on how to contribute to this project.

Here are just some things I think might be worth having

* Getting started documentation
* How can we work on UI components in this repo? (e.g. a local environment/Storybook?)

**Comment by NickDunas** [comment]
I have started writing the contribution guide multiple times, but I keep postponing it trying to add more new features first. 

Hopefully next week I will find some time to finish with it.

I will let you know once I do!

**Comment by NickDunas** [comment]
@tjcafferkey I finally managed to write a contribution guide. You can check it out here: https://github.com/rewindui/rewindui/blob/main/CONTRIBUTING.md

Let me know if it is as clear as it should!

**Reply by tjcafferkey** [reply]
On first glance it looks great, the only thing obvious that is missing is information about testing, and running them etc. But it's certainly a good start. Thank you!

As I use the library more in my evenings, I am sure I will contribute more so will revisit these docs and open some PRs for them if I think I can help improve them in any way.

**Reply by boweihan** [reply]
Added a small note on testing here - https://github.com/rewindui/rewindui/pull/77.
